django/template/defaulttags.py

```diff
@@ -1194,18 +1194,18 @@ def querystring(context, query_dict=None, **kwargs):
     """
     if query_dict is None:
         query_dict = context.request.GET
-    query_dict = query_dict.copy()
+    query_string_dict = query_dict.copy()
```

Comment:
To avoid confusion for future readers, I would use a different name, such as `params`.

Reply:
Updated to `params`, naming is hard :sweat_smile: